catalog/resource_system_schema.go

```diff
@@ -93,10 +93,21 @@ func ResourceSystemSchema() common.Resource {
 					if err != nil {
 						return err
 					}
+
+					// only track enabled schemas inline with rest of module
+					if schema.State != catalog.SystemSchemaInfoStateEnableCompleted &&
```

Comment:
We also need to handle NotAvailable - these are schemas that you can't enable anymore

Reply:
What should happen differently in this case? Currently (with this PR), it releases it from state and then assumedly fails when the Create function is called in the next run.

Reply:
@Waiwait I think that we need to exclude `UNAVAILABLE` status as well.

Reply:
Okay, I get it, thanks. Have added.